### PR TITLE
Add tests for KTable

### PR DESCRIPTION
**Pull request by ZiyaoWei**
Master Issue: #1266

### Motivation

#1266 reports that KTable doesn't work for a KoP enabled cluster. It turns out while we have tests for `GlobalKTable` we don't have tests for `KTable`.

### Modifications

Add tests similar to `GlobalKTableTest` but for `KTable`. The tests pass which means #1266 might have been caused by something external to KoP.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

This change added tests and can be verified as follows:

  - Added KTableTest

### Documentation

Need to update docs? 

- [x] `no-need-doc` 
  Test only

### CODE REVIEW

**Comment by BewareMyPower** [comment]
The license is wrong, please fix it first.

File: tests/src/test/java/io/streamnative/pulsar/handlers/kop/streams/KTableTest.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    @Override
+    protected Class<?> getValueSerdeClass() {
+        return null;
+    }
+
+    @Test(timeOut = 60000)
+    public void kTableReceivesAndUpdatesValues() throws Exception {
+        produceInitialTableValues();
+        table.mapValues((k, v) -> {
+            System.out.println(v);
```
[the remark this answers is not in the document]
Comment (ZiyaoWei) [reply tests/src/test/java/io/streamnative/pulsar/handlers/kop/streams/KTableTest.java:81]: Oops, thanks for catching!

**Comment by BewareMyPower** [comment]
ping @ZiyaoWei

**Review by eolivelli — APPROVED** [review]
LGTM

**Review by BewareMyPower — APPROVED** [review]

**Pull request merged**